packages/http-client-csharp/generator/Microsoft.Generator.CSharp.Shared/src/StringExtensions.cs

```diff
@@ -1,17 +1,14 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-using System;
-using System.Collections.Generic;
 using System.Diagnostics.CodeAnalysis;
-using System.Linq;
 using System.Text;
 using System.Text.RegularExpressions;
 using Microsoft.CodeAnalysis.CSharp;
 
-namespace Microsoft.Generator.CSharp
+namespace Microsoft.Generator.CSharp.Shared
 {
-    internal static class StringExtensions
+    public static class StringExtensions
```

Comment:
this should not be public, if you implement as shared source as I describe this won't be needed.